contributing_to_docs/term_glossary.adoc

```diff
@@ -514,6 +521,15 @@ Kubernetes API object that holds secret data of a certain type.
 
 See link:https://kubernetes.io/docs/concepts/configuration/secret/[Secrets - Kubernetes].
 
+''''
+=== security context constraints
```

Comment:
Should we mention explicitly the acronym SCC (of which we've got 150+ hits in the repo), or is allowance assumed? I see we've called out some abbreviations already, like CRD.

Reply:
Great idea, I'll add it